src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx

```diff
@@ -285,7 +285,7 @@ class ContactInformationField extends React.Component {
               fieldName,
               data: this.props.data,
               showSMSCheckbox: this.props.showSMSCheckbox,
-              editViewData: this.props.editViewData,
+              modalData: this.props.editViewData,
```

Comment:
oops! `editViewData` is not the option the helper is looking for :) https://github.com/department-of-veterans-affairs/vets-website/blob/4d43371444e438680187fbbafac1a9e6c27bdba4/src/applications/personalization/profile/util/contact-information/formValues.js#L37